nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/PutAzureBlobStorage_v12.java

```diff
@@ -231,6 +231,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             getLogger().error("Failed to create blob on Azure Blob Storage", e);
             flowFile = session.penalize(flowFile);
             session.transfer(flowFile, REL_FAILURE);
+            session.getProvenanceReporter().send(flowFile, "/error/" + blobName);
```

Comment:
The general purpose of provenance event reporting is to track successful operations as opposed to errors, so creating this event does not seem like the right approach. Is there a particular reason for this addition?

Reply:
That was something I forgot to remove. Thanks!